package.json

```diff
@@ -140,6 +152,35 @@
         "messages/**/*.md"
       ],
       "output": []
+    },
+    "test:command-reference": {
+      "command": "node --loader ts-node/esm --no-warnings=ExperimentalWarning \"./bin/dev.js\" commandreference:generate --erroronwarnings",
+      "files": [
+        "src/**/*.ts",
+        "messages/**",
+        "package.json"
+      ],
+      "output": [
+        "tmp/root"
+      ]
+    },
+    "test:deprecation-policy": {
+      "command": "node --loader ts-node/esm --no-warnings=ExperimentalWarning \"./bin/dev.js\" snapshot:compare",
+      "files": [
+        "src/**/*.ts"
+      ],
+      "output": [],
+      "dependencies": [
+        "compile"
+      ]
+    },
+    "test:json-schema": {
+      "command": "node --loader ts-node/esm --no-warnings=ExperimentalWarning \"./bin/dev.js\" schema:compare",
+      "files": [
+        "src/**/*.ts",
+        "schemas"
+      ],
+      "output": []
```

Comment:
this isn't running, also not supported if we use `single`?
https://github.com/oclif/plugin-command-snapshot/blob/65d0077d1aa30acd2ab921ae672cf4d2e156ade4/src/commands/schema/compare.ts#L48

Reply:
Yeah, plugin-command-snapshot won't be able to do the schemas or snapshot for a single CLI